armory/datasets/context.py

```diff
@@ -0,0 +1,34 @@
+import tensorflow as tf
+
+
+class ImageContext:
```

Comment:
I don't believe we'd actually be using any of the info in contexts except for `frame_rate`. We used to use the context objects for preprocessing but we aren't in the new setup. 

To my understanding, the issue here is that exporters need the dataset frame rate, and there's no way to access that from the dataset object. Is that correct? If that's so, we should make this piece as minimal as possible

Reply:
Yes you are correct that is the issue; I can reduce this down to be solely VideoContext containing frame-rate; I was unsure if other scenarios would require this context but I can selectively re-add as necessary.

Reply:
I think audio has sample_rate which is needed by the audio exporter for the same reason, see #1803

Reply:
Addressed by 0bdb3ec3ce074b5e33d590ea83ef7e0fd29da408. AudioContext can be added in a separate PR.